src/DebugBar/DebugBar.php

```diff
@@ -204,14 +204,27 @@ public function getHttpDriver()
      */
     public function collect()
     {
-        $this->data = array(
-            '__meta' => array(
-                'id' => $this->getCurrentRequestId(),
-                'datetime' => date('Y-m-d H:i:s'),
-                'utime' => microtime(true),
+        if (php_sapi_name() === 'cli') {
+            $request_variables = array(
+                'method' => 'CLI',
+                'uri' => isset($_SERVER['SCRIPT_FILENAME']) ? realpath($_SERVER['SCRIPT_FILENAME']) : null,
+                'ip' => gethostbyname(gethostname())
```

Comment:
Thanks - good catch!  You're right, it looks like it could fail, though I imagine it's quite rare/unlikely.  See the new fix-up commit.